src/node/cli.ts

```diff
@@ -126,6 +131,29 @@ async function resolveOptions(mode: string) {
     )
   }
 
+  argv = makeJsxObject(argv)
+
+  const userConfig = await resolveConfig(
+    argv.mode || defaultMode,
+    argv.config || argv.c
+  )
+  if (userConfig) {
+    return {
+      ...userConfig,
+      ...argv // cli options take higher priority
+    }
+  }
+
+  return argv
+}
+
+function makeJsxObject(argv: any) {
+  if (argv['jsxFactory']) {
+    ;(argv.jsx || (argv.jsx = {})).factory = argv['jsx-factory']
```

Comment:
Is using both `argv['jsxFactory']` and `argv['jsx-factory']` intentional?

Same with `jsx-fragment` and `jsxFragment`.

Reply:
I fixed it